src/wallet/rpcwallet.cpp

```diff
@@ -3220,6 +3220,7 @@ static UniValue bumpfee(const JSONRPCRequest& request)
             "                         so the new transaction will not be explicitly bip-125 replaceable (though it may\n"
             "                         still be replaceable in practice, for example if it has unconfirmed ancestors which\n"
             "                         are replaceable)."},
+                            {"changeAddress", RPCArg::Type::STR_HEX, /* opt */ true, /* default_val */ "", "The change address to pay the fee bump"},
```

Comment:
please follow the developer guidelines for naming new arguments:
```
 - Argument naming: use snake case `fee_delta` (and not, e.g. camel case `feeDelta`)
```

Reply:
err, bad paste.. ty